config.py

```diff
@@ -192,7 +192,7 @@ class Development(Config):
         'TRANSACTION_ISOLATION': False
     }
 
-    JIRA_FEATURES = True
+    JIRA_FEATURES = False
```

Comment:
shouldn't change this

Reply:
This came in when you pushed up changes to test fixtures.